ci-operator/step-registry/ipi/conf/openstack/ipi-conf-openstack-ref.yaml

```diff
@@ -6,7 +6,19 @@ ref:
     requests:
       cpu: 10m
       memory: 100Mi
+  credentials:
+  - namespace: test-credentials
+    name: cluster-secrets-openstack-vexxhost
+    mount_path: /var/run/cluster-secrets-openstack-vexxhost
+  - namespace: test-credentials
+    name: cluster-secrets-openstack-moc
+    mount_path: /var/run/cluster-secrets-openstack-moc
+  - namespace: test-credentials
+    name: cluster-secrets-openstack
+    mount_path: /var/run/.config/openstack
```

Comment:
It would be really nice if /var/run is the $HOME directory of the user executing the script, since we would not even need a OS_CLIENT_CONFIG_FILE anymore, but just to be safe.